qemu/tests/usb_common.py

```diff
@@ -129,6 +129,14 @@ def _verify_guest_usb():
                 return False
         return True
 
+    if params.get("os_type") == "linux":
+        logging.info("checking if there is I/O error in dmesg")
+        output = session.cmd_output("dmesg | grep usb",
```

Comment:
Talked with @maxujun, both uppercase and lowercase usb/USB should be considered.
And the same for the 'error' in line 137.

Reply:
@ngu-niny @maxujun  updated, thanks